src/component/mxgraph/shape/event-shapes.ts

```diff
@@ -52,6 +52,19 @@ abstract class EventShape extends mxEllipse {
       ShapeBpmnEventKind.COMPENSATION,
       (paintParameter: PaintParameter) => this.iconPainter.paintCompensationIcon({ ...paintParameter, ratioFromParent: 0.7, icon: { ...paintParameter.icon, strokeWidth: 1.5 } }),
     ],
+    [
+      ShapeBpmnEventKind.CANCEL,
+      (paintParameter: PaintParameter) =>
+        this.iconPainter.paintCancelCrossIcon({
+          ...paintParameter,
+          setIconOrigin: (canvas: BpmnCanvas) => canvas.setIconOriginToShapeTopLeft(2, 8),
```

Comment:
There is a little space on the right (and not on the left).
![image](https://user-images.githubusercontent.com/4921914/96597054-aa618900-12ed-11eb-9e92-8c88dc189cf4.png)

Reply:
I think it's more aligned with this:
```suggestion
          setIconOrigin: (canvas: BpmnCanvas) => canvas.setIconOriginToShapeTopLeft(2, 9),
```

**Result**
![image](https://user-images.githubusercontent.com/4921914/96597312-f90f2300-12ed-11eb-9402-9ae643426c79.png)

Reply:
> There is a little space on the right (and not on the left).
> ![image](https://user-images.githubusercontent.com/4921914/96597054-aa618900-12ed-11eb-9e92-8c88dc189cf4.png)

there is something wrong as it renders well with file: model-complete-semantic.bpmn and badly with events.bpmn

Reply:
Maybe the size of the shape of the event are not the the same in the 2 files.
If it's the case, there is a problem with the positionning of the icon. We need to position it according to the size of the shape, not with a margin.

Reply:
fixed!